code/modules/mob/living/simple_animal/friendly/rabbit.dm

```diff
@@ -0,0 +1,78 @@
+//Good ol' rabbits
+
+/mob/living/simple_animal/rabbit
```

Comment:
proper dmdoc, short desc of what they do

Reply:
that means the // comment needs to gain one more / and be right above the type- it is missing a general gist

Reply:
I looked at what other simple_mobs did for that and put that in the latest commit.